### PR TITLE
Improve cd auto complete

### PR DESCRIPTION
**Pull request by lborguetti**
Now, when you try complete with cd `nashcomplete` offer only directories. And Small fixes made `nashcomplete` user-friendliness.

### CODE REVIEW

File: files.sh [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 	var dirpath = ""
+	var findArgs = ()
+	var fname = ""
+	var last = ""
+	var lastpart = ""
+	var partsz = ""
+	var status = ""
+
+	partsz           <= len($parts)
+	last, status     <= expr $partsz - 1
```
Comment (katcipis) [inline files.sh]: if the objective is to ignore the status and just dont crash you can use **_** to make the intention explicit (I was looking to where you use status but it seems that you dont).

**Review by katcipis — APPROVED** [review]
LGTM, thanks dude :+1:

File: files.sh (hunk 1 above)
Comment (lborguetti) [reply files.sh]: Yes, your right ;)

Comment (lborguetti) [reply files.sh]: done! fix that!

**Pull request merged**